gcp/api/test_server.py

```diff
@@ -139,7 +139,7 @@ def start_esp(port, backend_port, credential_path, log_path):
         '--non_gcp',
         '--enable_debug',
         '--transcoding_preserve_proto_field_names',
-        '--envoy_connection_buffer_limit_bytes=10485760',
+        '--envoy_connection_buffer_limit_bytes=104857600',
```

Comment:
is there some prod config we have to change here too?

Reply:
Yes, just discovered this when running on staging, we need to update the terraform config as well, I'll make a follow-up PR after testing on staging.